support/loggers/loggers.go

```diff
@@ -0,0 +1,53 @@
+package loggers // I didn't call the packacke "logging" because the go-loggeing framework's package is "logging" so we'll need to be different when we switch
+
+import (
+	"log"
```

Comment:
we should not import `log` here.
this file should only contain the Logger interface.

BasicLogger should be in it's own file `support/logger/basicLogger.go` or `support/logger/nativeLogger.go`